### DISCUSSION TITLE
Is prisma has some immutable capabilities?

### DISCUSSION THREAD

**Discussion by firehist** [prisma/prisma#10768] [Q&A]
I wondering if there is 'best way" or "built-in" functionalities to set up an immutable system in our databases.

I mean by an immutable system, a way to store the full history of a resource with:
- No update, only insert;
- A version number bumped;
- Taking care of dependencies to get the latest version from a resource & child;
- etc.

I am really interested in some insights to manage that with Prisma.

I already did the job by hand on a more direct stack API (Scala) <=> DB, but wondering if this can be easily usable with this astonishing project!

Thanks a lot!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
